LinkLiteAgent/.gitignore

```diff
@@ -0,0 +1,87 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
+
+# Distribution / packaging
```

Comment:
I would suggest as much as possible / suitable is moved to the root repo gitignore, to avoid duplication when multiple python projects are added (libs, other agent apps etc.)